crates/sui-sdk/src/crypto.rs

```diff
@@ -135,6 +135,9 @@ impl SuiKeystoreSigner {
 
 impl signature::Signer<Signature> for SuiKeystoreSigner {
     fn try_sign(&self, msg: &[u8]) -> Result<Signature, signature::Error> {
-        self.keystore.read().unwrap().sign(&self.address, msg)
```

Comment:
This should be a panic, as the only time this unwrap would trigger would be on a poisoned locked which we have no way of recovering from

Reply:
Fair enough, my panic aversion should not extend to client binaries.